airbyte-connector-builder-server/connector_builder/generated/models/http_request.py

```diff
@@ -20,11 +20,13 @@ class HttpRequest(BaseModel):
         parameters: The parameters of this HttpRequest [Optional].
         body: The body of this HttpRequest [Optional].
         headers: The headers of this HttpRequest [Optional].
+        http_method: The http_method of this HttpRequest [Optional].
     """
 
     url: str
     parameters: Optional[Dict[str, Any]] = None
     body: Optional[Dict[str, Any]] = None
     headers: Optional[Dict[str, Any]] = None
+    http_method: Optional[str] = None
```

Comment:
should we make this a required field? I can't imagine a request without an HTTP method

Reply:
Makes sense to me @girarda. I made this change. I'm still using `.get` on the request [(with a default `""`)](https://github.com/airbytehq/airbyte/pull/20109/files#diff-9d15ecb8c45b75149182f5a9ec28f04425f591e7709f94195aff524a0f53d7afR182) - I assume that if there is a case where `http_method` is missing we don't want to throw an error in the UI. But let me know if you think otherwise.